src/cgame/cg_buildable.c

```diff
@@ -677,7 +687,8 @@ void CG_GhostBuildable(buildable_t buildable)
 
 	CG_PositionAndOrientateBuildable(ps->viewangles, entity_origin,
 					 tr.plane.normal, ps->clientNum, mins,
-					 maxs, ent.axis, ent.origin);
+					 maxs, ent.axis, ent.origin, 
+					 BG_FindTeamForBuildable(buildable) == BIT_ALIENS ? qtrue : qfalse );
```

Comment:
`a == b ? true : false` is redundant.

Also I'd pass `buildable` itself to the function, so we can do tweaks per-building rather than per-team.